src/commands/index.ts

```diff
@@ -0,0 +1,95 @@
+import * as vscode from 'vscode';
+import { getConfig } from '../utils/vscode-extensions';
```

Comment:
I'm leaving it this way for the moment. We can change this in a later PR if we feel like it.